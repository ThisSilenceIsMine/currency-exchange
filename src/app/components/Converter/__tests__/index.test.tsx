import * as React from 'react';
import { render } from '@testing-library/react';

import { Converter } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<Converter  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Converter />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
