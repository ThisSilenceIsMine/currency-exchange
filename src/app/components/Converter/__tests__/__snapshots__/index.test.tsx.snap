// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Converter  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  width: 40%;
  margin: 0 auto;
  margin-top: 10em;
}

.c1 {
  background-color: white;
  border: 1px solid black;
  border-radius: 0.3em;
  padding: 0.5em;
  width: 66%;
}

.c2 {
  background-color: white;
  border: 1px solid black;
  border-radius: 0.3em;
  padding: 0.5em;
  width: 33%;
}

<div
  class="c0"
>
  <input
    class="c1"
    name="from"
    type="text"
    width="66%"
  />
  <input
    class="c2"
    name="result"
    type="text"
    width="33%"
  />
</div>
`;
